module/addLocal.js

```diff
@@ -0,0 +1,32 @@
+export default class Data {
+    static #books = [];
+
+    static #checkData() {
+      return localStorage.getItem('Books');
+    }
+
+    static getData() {
+      if (Data.#checkData()) {
+        Data.#books = JSON.parse(localStorage.getItem('Books'));
+      }
+      return Data.#books;
+    }
+
+    static saveData(book) {
+      if (Data.#checkData()) {
+        Data.#books = JSON.parse(localStorage.getItem('Books'));
+      }
+
+      Data.#books.push(book);
+      localStorage.setItem('Books', JSON.stringify(Data.#books));
+    }
+
+    static removeData(bookId) {
+      if (Data.#checkData()) {
+        let books = Data.#books;
+        books = books.filter((book) => book.id !== bookId);
+        localStorage.setItem('Books', JSON.stringify(books));
+        Data.#books = books;
+      }
+    }
+}
```

Comment:
- [ ] Good job so far your code looks clean and awesome 👏 However as part of the requirements we should stick with ES6 so we need always to use arrow functions, I noticed that all of your classes are normal functions, and not arrow functions, example : 

```JS
  // Normal Function
   static saveData(book) { ... }
  // Arrow Function
    static saveData = (book) => { ... }
```

kindly fix all of your normal functions, including the ones in `addLocal.js`, `errorCheck.js`, `mainClass.js`, `showBook.js` 🙏